From 900aff6eba1547fb3ebc09aa756f0d50704ca885 Mon Sep 17 00:00:00 2001
From: Matthias Runge <mrunge@redhat.com>
Date: Tue, 17 Jun 2014 09:04:42 +0200
Subject: [PATCH] Re-enable offline compression

---
 openstack_dashboard/settings.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/openstack_dashboard/settings.py b/openstack_dashboard/settings.py
index 01e3308..7d5bcb9 100644
--- a/openstack_dashboard/settings.py
+++ b/openstack_dashboard/settings.py
@@ -146,7 +146,7 @@ COMPRESS_ENABLED = True
 COMPRESS_OUTPUT_DIR = 'dashboard'
 COMPRESS_CSS_HASHING_METHOD = 'hash'
 COMPRESS_PARSER = 'compressor.parser.HtmlParser'
-COMPRESS_OFFLINE = False
+COMPRESS_OFFLINE = True
 
 INSTALLED_APPS = [
     'openstack_dashboard',
